### ISSUE TITLE
Broken build when upgrading to 74.0.3729.169

### ISSUE THREAD

**Issue by saltama** [ungoogled-software/ungoogled-chromium-macos#11]
Right now, building the latest release tagged on the main project results in a `Only source, header, and object files belong in the sources of a static_library` error.
It looks like the culprits are some generated python files but adding them to `pruning_list.txt` doesn't help.
Haven't solved this yet, if someone has any ideas, feel free to comment.

```
+ ./tools/gn/bootstrap/bootstrap.py -o out/Default/gn --skip-generate-buildfiles
ninja: Entering directory `ungoogled-chromium-macos/build/src/out/Release/gn_build'
[178/178] LINK gn
+ ./out/Default/gn gen out/Default --fail-on-unused-args
ERROR at //third_party/protobuf/proto_library.gni:369:15: Only source, header, and object files belong in the sources of a static_library. //out/Default/pyproto/google_apis/gcm/protocol/mcs_pb2.py is not one of the valid types.
    sources = get_target_outputs(":$action_name")
              ^---------------------------------
See //google_apis/gcm/BUILD.gn:78:1: whence it was called.
proto_library("proto") {
^-----------------------
See //BUILD.gn:81:7: which caused the file to be included.
      "//google_apis/gcm:gcm_unit_tests",
      ^---------------------------------
```

**Comment by Eloston** [comment]
We don't really want any code from `//google_apis`. The first place to look would probably be in `//google_apis/gcm/BUILD.gn` and/or `//BUILD.gn` at those lines indicated to see what GN config we will need to modify.

**Comment by Zoraver** [comment]
I ran into a similar issue when trying to build 74.0.3729.169 for Linux, which I believe to be caused by [this bug](https://bugs.chromium.org/p/chromium/issues/detail?id=966066&q=&colspec=ID%20Pri%20M%20Stars%20ReleaseBlock%20Component%20Status%20Owner%20Summary%20OS%20Modified) upstream. I got around it by downloading the source for 74.0.3729.157 and replacing the gn source folder in 74.0.3729.169 with the gn source folder from 74.0.3729.157. It isn't ideal, which is why I haven't made a pull request to update the portablelinux repo to 74.0.3729.169, but I was able to produce a working build.
